src/com/lftechnology/java/training/alina/basic/smallestnumber/SmallestPositiveNumber.java

```diff
@@ -5,74 +5,69 @@
 import java.util.logging.Logger;
 
 /**
- * 2520 is the smallest number that can be divided by each of the numbers from 1
- * to 10 without any remainder. What is the smallest positive number that is
- * evenly divisible by all of the numbers from 1 to 20?
+ * 2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder. What is the smallest positive
+ * number that is evenly divisible by all of the numbers from 1 to 20?
  * 
  * @author Alina Shakya <alinashakya@lftechnology.com>
  *
  */
 public class SmallestPositiveNumber {
 
-	private static final Logger LOGGER = Logger
-			.getLogger(SmallestPositiveNumber.class.getName());
+    private static final Logger LOGGER = Logger.getLogger(SmallestPositiveNumber.class.getName());
 
-	public static void main(String[] args) {
-		try (Scanner scanner = new Scanner(System.in)) {
-			int rangeLimit;
-			LOGGER.info("Enter the maximum range number > ");
-			rangeLimit = scanner.nextInt();
-			Long smallestNumber = displaySmallestNumber(rangeLimit);
-			LOGGER.log(
-					Level.INFO,
-					"The smallest positive number that is evenly divisible by all of the numbers from 1 to {0} is : {1}",
-					new Object[] { rangeLimit, smallestNumber.toString() });
-		} catch (Exception e) {
-			LOGGER.log(Level.WARNING, "Exception Message : {0}",
-					new Object[] { e.getMessage() });
-		}
-	}
+    public static void main(String[] args) {
+        try (Scanner scanner = new Scanner(System.in)) {
+            int rangeLimit;
+            LOGGER.info("Enter the maximum range number > ");
+            rangeLimit = scanner.nextInt();
+            Long smallestNumber = displaySmallestNumber(rangeLimit);
+            LOGGER.log(Level.INFO, "The smallest positive number that is evenly divisible by all of the numbers from 1 to {0} is : {1}",
+                    new Object[] { rangeLimit, smallestNumber.toString() });
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)